app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -140,6 +140,10 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
 
         lifecycle.addObserver(webExtensionPopupFeature)
         StartupTimeline.onActivityCreateEndHome(this)
+
+        if(shouldStartInRecentsScreen(intent)) {
+            moveTaskToBack(true)
```

Comment:
Please remove the `START_IN_RECENTS_SCREEN ` extra once it's used, otherwise when you click on the notification while in background, when you come back to the app, every time the activity is recreated (as example, you push the private mode button), the task will be moved to back.
`intent.removeExtra(START_IN_RECENTS_SCREEN)`